ocs_ci/helpers/helpers.py

```diff
@@ -2961,3 +2962,43 @@ def get_mon_pdb():
     disruptions_allowed = pdb_obj.get().get("status").get("disruptionsAllowed")
     min_available_mon = pdb_obj.get().get("spec").get("minAvailable")
     return disruptions_allowed, min_available_mon
+
+
+def check_pvs_present_for_ocs_expansion(sc=constants.LOCALSTORAGE_SC):
```

Comment:
I have created this file https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/resources/pv.py in my previous PR. This function will be more suitable in the file above since it's for the pv functions.